src/fetch_agent/fetch_agent/agent.py

```diff
@@ -44,6 +45,8 @@ async def startup_event(ctx: Context):
     # run function in background so agent can fully start while registering
     asyncio.ensure_future(register_at_registry(ctx, agent))
 
+    queuing_system = QueuingSystem(reservations=[])
+
     properties = PropertyData(
         open_time_frames=[(0, 0), (0, 0)],
```

Comment:
Should be
`
queuing_system.get_open_time_frames()
`
Or whatever the function is called